tests/test_FiberFlat.py

```diff
@@ -0,0 +1,67 @@
+import sys
+import unittest
+import pickle
+
+import numpy as np
+
+import lsst.utils.tests
+import lsst.pex.exceptions
+import lsst.geom
+import lsst.afw.image
+import lsst.afw.image.testUtils
+
+import pfs.drp.stella as drpStella
+import lsst.afw.image as afwImage
+from pfs.drp.stella.constructFiberFlatTask import ConstructFiberFlatTask
+
+
+class FiberFlatTestCase(lsst.utils.tests.TestCase):
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def testExtractNonOptimal(self):
+        """ Tests the ConstructFiberFlatTask.correctLowVarianceImage method
+        """
+        width = 4096
+        height = 4176
```

Comment:
Do you need such a large image? That chews memory and CPU unnecessarily, and makes tests run slower than they need to.

Reply:
Reduced image size. Fixed.